.github/workflows/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10.x', '12.x', '14.x' ]
+    name: Node ${{ matrix.node }}
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        git remote set-branches --add origin master
+        git fetch
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - name: Install Dependencies
+      run: npm ci
+      env:
+        NODE_ENV: development
+    - name: Unit Tests
+      run: ./node_modules/.bin/jest
```

Comment:
How do we keep all of these in sync with what is in `npm test`?

Reply:
@JamesSingleton and I discussed over slack and decided it is fine to take this considering how infrequently test scripts are added, and the point that any opener or reviewer of a new test script would be particularly mindful of the tests that are being run